src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx

```diff
@@ -178,7 +178,7 @@ export default class ResultTable extends React.Component {
                 rowCount={data.length + 1}
                 columnCount={colCount}
                 fixedRowCount={1}
-                rowHeight={40}
+                rowHeight={41} // Add 1px for border
```

Comment:
Does this build? This shouldn't be a valid comment

Reply:
it builds, and my editor marks it as a valid comment :(